war/src/main/scss/components/_icons.scss

```diff
@@ -127,7 +127,5 @@
 }
 
 .pulse-animation {
-  svg {
```

Comment:
Seems unrelated?

Reply:
Previously the `svg` was unnecessarily wrapped in a `span`, hence the need to specify `svg`. Seeing as the span has been removed this selector will no longer work - so I've updated it.